src/lib.js

```diff
@@ -466,7 +466,11 @@ var program,
         this.initialize();
         this.requestUrl = '';
 
-        var sanitizedArgs = this.sanitizeArgs(curlString),
+        // [Github #8843] - RegEx to fix malformed cURLs with unquoted multi-param URLs
+        const multiParamUrlRegEx = /[^ ]+\.[^ ]+&[^ ]+/gm;
```

Comment:
What will happen if we have a URL like this > 
`https://postman-echo.com/get?foo1=bar1&` 

@SahilChoudhary22